marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -554,6 +572,80 @@ const Flows: React.FC<Props> = ({
     }
   };
 
+  const getLocalStorageDataUser = () => {
+
+    if (getUserPreferencesLS()) {
```

Comment:
Same suggestion here. Let's get the return value of the function getUserPreferencesLS() before line 577 here and then use it further down inside the if logic.